TeamCode/src/main/java/org/firstinspires/ftc/teamcode/HardwareComponents/Drivetrain.java

```diff
@@ -0,0 +1,99 @@
+package org.firstinspires.ftc.teamcode.HardwareComponents;
+
+
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+import org.firstinspires.ftc.teamcode.Movement.AutonomousRoute;
+import org.firstinspires.ftc.teamcode.HardwareComponents.Motors.EncodedMotor;
+import org.firstinspires.ftc.teamcode.Movement.Route;
+
+import static android.os.SystemClock.sleep;
+
+public class Drivetrain {
+    static final double ENCODER_TICKS = 1440;
+    static final double GEAR_REDUCTION = 2.0;
+    static final double WHEEL_DIAMETER = 5.0;
+    static final double TICKS_PER_INCH = (ENCODER_TICKS * GEAR_REDUCTION) / (WHEEL_DIAMETER * Math.PI);
+
+    public final EncodedMotor leftDriveMotor;
+    public final EncodedMotor rightDriveMotor;
+    public double currentLeftPower, currentRightPower;
+
+    private ElapsedTime runtime = new ElapsedTime();
+
+
+    /**
+     * Drivetrain consists of left and right drive motors.
+     * For accurate paths, ensure WHEEL_DIAMETER and GEAR_REDUCTION are
+     * set correctly.
+     * @param leftDrive DcMotor for left drive.
+     * @param rightDrive DcMotor for right drive.
+     * @param isDriverControl Whether autonomous or driver controlled.
+     */
+    public Drivetrain(DcMotor leftDrive, DcMotor rightDrive, boolean isDriverControl) {
+        leftDriveMotor = new EncodedMotor(leftDrive);
+        rightDriveMotor = new EncodedMotor(rightDrive);
+        leftDriveMotor.setReverse();
+
+        if (isDriverControl) {
+            leftDriveMotor.setMode(DcMotor.RunMode.RUN_WITHOUT_ENCODER);
+            rightDriveMotor.setMode(DcMotor.RunMode.RUN_WITHOUT_ENCODER);
+        } else {
+            leftDriveMotor.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+            rightDriveMotor.setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+            leftDriveMotor.minPower = 0.2;
+            rightDriveMotor.minPower = 0.2;
+        }
+        currentLeftPower = currentRightPower = 0;
```

Comment:
I generally shun the multiple assignment syntax because it can be rather confusing, especially to newbies.

Recommend using 
```
currentLeftPower = 0; 
currentRightPower = 0;
```

or at least
` currentLeftPower = 0, currentRightPower = 0;`

Reply:
Probably go for the prior. Funny... when I was writing this with the kids, I said something to the effect of, "Programmers are lazy and they hate writing code. Here's another shortcut"